packages/emotion/src/sheet.js

```diff
@@ -56,7 +56,9 @@ export default class StyleSheet {
       throw new Error('already injected!')
     }
     if (this.isBrowser) {
-      this.tags[0] = makeStyleTag()
+      // If an existing style tag with a nonce attribute is present, use that before attempting to create a new tag.
+      const tagWithNonce = document.querySelector('style[nonce]')
```

Comment:
I'm not sure about the `style[nonce]` selector since it'll target any style element with the nonce attribute, maybe `style[nonce][data-emotion]`?

Reply:
Good idea @mitchellhamilton

Reply:
@mitchellhamilton That's a *much* better idea!